src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -861,8 +861,8 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		}
 	}
 
-	invokeWithinContext<T>(fn: (accessor: ServicesAccessor) => T): T {
-		return this.scopedInstantiationService.invokeFunction(fn);
+	getInternalContextKeyService(): IContextKeyService {
+		return this.scopedInstantiationService.invokeFunction(accessor => accessor.get(IContextKeyService));
```

Comment:
@roblourens why not keep the scoped context key service around to return directly? Could even just be a getter.